examples/ftp.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Example for initiating a ftp backend.
+
+use anyhow::Result;
+use log::info;
+use opendal::services::ftp;
+use opendal::services::ftp::Builder;
+use opendal::Operator;
+use std::env;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    if env::var("RUST_LOG").is_err() {
+        env::set_var("RUST_LOG", "debug");
+    }
+    env_logger::init();
+
+    println!(
+        r#"OpenDAL ftp Example.
+
+Available Environment Values:
+    - OPENDAL_FTP_ENDPOINT=endpoint     # required  
+    - OPENDAL_FTP_Port=port      # default with 21
```

Comment:
I prefer merging `port` into the endpoint.

- Keep the same style with other services.
- Easier for using.